### DISCUSSION TITLE
Add Fun and Motivational Tips to Beatmap Loading Screens

### DISCUSSION THREAD

**Discussion by DarkGradient** [ppy/osu#31090] [Ideas] [answered]
**[ NOT FOR MAIN MENU ]**
Sometimes I wonder why there are no tips displayed when loading a beatmap to play. These tips could be incredibly helpful for players. For example, they could include motivational messages, reminders to take breaks if someone has been playing for too long, or useful advice about avoiding mindblocks (not just in the main menu!).

Such tips could also help players deal with frustration, like when rank is taken away, or performance points (pp) aren’t awarded. By displaying encouraging or informative messages, the game could distract from negative emotions and help refocus on enjoying the experience.

Here are some example tips that could be displayed:

- "Remember, playing osu! is about having fun! Don’t stress over ranks too much."
- "Struggling with a beatmap? Try slowing it down in practice mode! Use NoFail, or DayCore mods."
- "Looks like you playing too long [_Insert time played, or retry count_]. Take a break to stretch and reset your focus."
- "Mindblocks are normal! Step away for a while or try a different beatmap."
- "Every fail is a step closer to mastery. Keep going!"

This small addition could support new players, help experienced ones stay engaged, and remind everyone that playing games is all about having fun.

_P.S.: Maybe this idea already exists, but I couldn’t find it, probably due to poor search phrasing._

**Comment by peppy** [comment, marked as answer]
Yeah I think we could probably show the main menu tips at player loader as well, or as you say, a different set of tips. Seems like a good usage of space to give people some motivation, or a laugh.
